src/views/Stake/__tests__/Stake.unit.test.tsx

```diff
@@ -1,8 +1,23 @@
+import mediaQuery from "css-mediaquery";
 import { ReactQueryProvider } from "src/lib/react-query";
 
 import { render, screen } from "../../../testUtils";
 import Stake from "../Stake";
 
+function createMatchMedia(width: any) {
```

Comment:
@brightiron it looks like this is a function that we can reuse again? Would it be worth moving it to `testHelpers`?

Reply:
yup that's fine! Just hadn't had a use case yet for it to be reused, but yes lets refactor it out in the next test